app/controllers/pets_controller.rb

```diff
@@ -5,6 +5,10 @@ class PetsController < ApplicationController
   # GET /pets.json
   def index
     @pets = Pet.all
+
+    if not @pets.nil?
+      @pets = @pets.sort { |x, y| -(x.submitted_at <=> y.submitted_at) }
```

Comment:
There is a potential problem sorting items within the application: you have to retrieve all items in order to be able to sort. 'All items' is very dangerous from the performance and memory consumption perspective. Remember 'All items' is always suspicious. Think ahead: how many items the system may handle during a year or two? Thousands? Millions? So, sorting using the database capability would be much more effective and scalable.  
P.S. @pets = Pets.all - is a smell. We should use paging here, returning limited number of records, never all. So, don't treat existing code as perfect. Don't hesitate to criticize it.